CTX.js

```diff
@@ -171,25 +100,23 @@ function contextObjectXMLToCOinS (text) {
 		Zotero.debug("Processing object: " + objects[i].textContent);
 		var pieces = [];
 		
-		
 		var version = ZU.xpathText(objects[i], './@version', ns)
 	
 		pieces.push("ctx_ver="+encodeURIComponent(version));
 		
-		var format = ZU.xpathText(objects[i], '//ctx:format', ns)
+		var format = ZU.xpathText(objects[i], './/ctx:format', ns)
 		// Now conert this to the corresponding Key/Encoded-Value format; see note below.
 		// Check if this is unknown; if it is, skip
 		if (format == "info:ofi/fmt:xml:xsd:unknown") {
-			Zotero.debug("Skipping object of type 'unknown'");
-			continue;
+			// use journalArticle as default value
+			format = "info:ofi/fmt:kev:mtx:journal";
```

Comment:
either "info:ofi/fmt:xml:xsd:journal" or don't look up this value in `mapXMLtoKEV` below